docs/how-to/debugging_env.rst

```diff
@@ -0,0 +1,94 @@
+.. list-table::
+    :header-rows: 1
+    :widths: 35,14,51
+
+    * - **Environment variable**
+      - **Default value**
+      - **Value**
+
+    * - | ``AMD_LOG_LEVEL``
+        | Enables HIP log on various level.
+      - ``0``
+      - | 0: Disable log.
+        | 1: Enables log on error level.
+        | 2: Enables log on warning and lower levels.
+        | 3: Enables log on information and lower levels.
+        | 4: Enables log on debug and lower levels.
```

Comment:
There is Debug extra log as well.

Reply:
Wow, this is new. Thanks:

https://github.com/ROCm/clr/blob/amd-staging/rocclr/utils/debug.hpp#L38-L45